brigade/static/scss/molecules/_card.scss

```diff
@@ -5,32 +5,119 @@
 //
 
 .card {
+  font-weight: inherit;
+
+  display: flex;
+  flex-direction: row;
+
+  margin: 15px 0;
+
   box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);
+
+  flex-wrap: wrap;
```

Comment:
Style nitpick: I've never separated CSS rules like this before -- and I'm uncertain that it will produce more readable code. But regardless, I think the most important thing is that we only have one way to do a thing, so whatever we do, we should pick a single way to organize these rules. Reading this code, it is not intuitive to me which order these rules go in -- e.g. why does "flex-wrap" go at the bottom but "flex-direction" in the middle?

I think a much more intuitive ordering system would be to just group all the rules together and alphabetize them.